elide-contrib/elide-dynamic-config-helpers/src/test/resources/validator/valid/security.hjson

```diff
@@ -0,0 +1,18 @@
+{
+    roles : [
+        admin
+        guest
+        <% role %>
+        ]
+    rules: [
+        {
+            type: filter
+            filter: company_id=in=${principal.companies}
```

Comment:
as per the latest schema, this value will not work.

Reply:
changed to filter enum